lib/bootloader_setup.pm

```diff
@@ -84,30 +84,39 @@ And of course the new entries have C<ima_policy=tcb> added to kernel parameters.
 sub add_custom_grub_entries {
     my $grub_param = get_var('GRUB_PARAM');
     return unless defined($grub_param);
+    my $script_old     = "/etc/grub.d/10_linux";
```

Comment:
in a fact it is menuentry, 'script' in a variable name looks confusing for me

Reply:
@asmorodskyi If you look inside, it's a shell script.